Makefile

```diff
@@ -112,6 +112,12 @@ GO_TEST_PACKAGES=./cmd/... ./pkg/...
 # Use an environment variable to allow CI to disable when cross-compiling.
 export CGO_ENABLED ?= 1
 
+# Specify OCP build tools image tag when building rpm with podman
+RPM_BUILDER_IMAGE_TAG := rhel-9-release-golang-1.20-openshift-4.15
+ifeq "$(strip $(shell uname -m))" "aarch64"
+	RPM_BUILDER_IMAGE_TAG := $(RPM_BUILDER_IMAGE_TAG).art-arm64
```

Comment:
Will the naming convention using this suffix always hold, or should we just use 2 full strings for clarity?